metadata_mapper/mappers/mapper.py

```diff
@@ -308,8 +313,8 @@ def copy_prop(self, prop, to_prop, skip_if_exists=[False]):
                     to_prop=["sourceResource/title"]
         """
 
-        src = prop[0].split('/')[1:]
-        dest = to_prop[0].split('/')[1:]
+        src = prop[0].split('/')[-1]
+        dest = to_prop[0].split('/')[-1]
```

Comment:
this update handles `to_prop = ["sourceResource/title"]` AND `to_prop = ["dataProvider"]`

(though obviously still unclear on what data provider means in the to_prop category, we'll have to wait till we harvest a collection that uses this enrichment with these parameters)